server/functions.lua

```diff
@@ -291,14 +293,22 @@ function QBCore.Functions.RemovePermission(source, permission)
         if IsPlayerAceAllowed(source, permission) then
             ExecuteCommand(('remove_principal player.%s qbcore.%s'):format(source, permission))
             QBCore.Commands.Refresh(source)
```

Comment:
(optional) consider extracting the duplicate code into a shared function by these two functions

Reply:
I'll re-_view_ later whenever I do the permission update